include/rapidjson/writer.h

```diff
@@ -352,7 +352,7 @@ class Writer {
         char* end = internal::dtoa(d, buffer, maxDecimalPlaces_);
         PutReserve(*os_, static_cast<size_t>(end - buffer));
         for (char* p = buffer; p != end; ++p)
-            PutUnsafe(*os_, static_cast<typename TargetEncoding::Ch>(*p));
+            PutUnsafe(*os_, static_cast<typename OutputStream::Ch>(*p));
```

Comment:
Yes. From the definition of `PutUnsafe()`:

~~~cpp
template<typename Stream>
inline void PutUnsafe(Stream& stream, typename Stream::Ch c) {
    stream.Put(c);
~~~

It should use the type in output stream. But I cannot think of any situation that these two character types are different.